hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -38,3 +38,43 @@ ANNOTATION: "Waived the assignment as the input is wired to a parameter"
 Block 1 "3318159292" "data_o = 39'(data_i);"
 INSTANCE: tb.dut.u_tl_adapter_rom.u_tlul_data_integ_enc_instr.u_data_gen
 Block 1 "3318159292" "data_o = 39'(data_i);"
+
+INSTANCE: tb.dut.u_tl_adapter_rom
+// wr_attr_error and wr_vld_error has dependence over same type of a_opcode. Thus we can't get
+// wr_attr_error alone.
+Condition 17 "3469950311" "(wr_attr_error | wr_vld_error | rd_vld_error | instr_error | tlul_error |
+                            intg_error) 1 -1" (7 "100000")
+
+// We can't get an instr_error alone without tlul_error. Reason is same that both bits depends upon
+// instruction type being invalid.
+Condition 17 "3469950311" "(wr_attr_error | wr_vld_error | rd_vld_error | instr_error | tlul_error |
+                            intg_error) 1 -1" (4 "000100")
+
+// For rspfifo_rdata.error, it is pushed in rspfifo_wdata.error which comes from rerror_i on L529
+// in tlul_adapter_sram.sv and rerror_i is wired to 0 in rom_ctrl.sv L203
+Condition 4 "3638058042" "(rspfifo_rdata.error | reqfifo_rdata.error) 1 -1" (3 "10")
+
+// reqfifo_rvalid implies d_valid. Read from both req and rsp fifos happened on the same cycle. So
+// expecting rspfifo_rvalid without d_valid is a non-occuring case.
+Condition 21 "3623514242" "(d_valid & rspfifo_rvalid & (reqfifo_rdata.op == OpRead)) 1 -1" (1 "011")
+
+// Condition 24 till 34 are excluded for the same reason. The requirements for the conditions are:
+// 1) reqfifo_valid --> 1
+// 2) rspfifo_valid --> 1
+// 3) reqfifo_rdata.op --> opread
+// 4) refifo_rdata.error --> 1
+// For condition 24 to 26, we don't expect a rspfifo_rvalid with a reqfifo_rdata.error. Thus it
+// restrict the case 10, 1, 11 and 01 for conditions 24 till 26 as non-ocurring. For condition 34,
+// d_error needs reqfifo_rvalid and reqfifo_rdata.error to get set which automatically set the
+// d_valid. So 01 for condition 24 is non-occuring.
+Condition 24 "1059982851" "(vld_rd_rsp & ((~d_error))) 1 -1" (2 "10")
+Condition 25 "2807788926" "((vld_rd_rsp && reqfifo_rdata.error) ? error_blanking_integ :
+                            (vld_rd_rsp ? rspfifo_rdata.data_intg :
+                            prim_secded_pkg::SecdedInv3932ZeroEcc)) 1 -1" (2 "1")
+Condition 26 "561780173" "(vld_rd_rsp && reqfifo_rdata.error) 1 -1" (3 "11")
+Condition 34 "201396280" "(d_valid && d_error) 1 -1" (1 "01")
+
+// When access is granted to the bus and rom respond to the bus request leads the rvalid_i signal
+// go high and reqfifo_rvalid gets set on that cycle until the read happens. Hence, 10 is
+// non-occuring for the conditional statement.
```

Comment:
I had to re-convince myself about why this was correct. I'm not sure the text here makes it particularly obvious. Maybe something like this instead?
```systemverilog
// It is impossible to see rvalid_i && !reqfifo_rvalid. If rvalid_i is true, the ROM is responding
// to a request that was sent on the previous cycle. Since it always responds in exactly one cycle,
// the request will not already have been popped from the request fifo, so reqfifo_rvalid must be
// true.
```